### ISSUE TITLE
On 2.1.67-beta, Studio Controls won't start due to a list index being out of range

### ISSUE THREAD

**Issue by CheetahPixie** [ovenwerks/studio-controls#59]
```
$ studio-controls 
install path: /usr

Config file: /home/pixie/.config/autojack/autojack.json
Traceback (most recent call last):
  File "/usr/bin/studio-controls", line 1840, in <module>
    us = StudioControls()
  File "/usr/bin/studio-controls", line 559, in __init__
    self.devdb.scan_dev(self.conf_db)
  File "/usr/bin/studio-controls", line 208, in scan_dev
    self.get_dev_info(conf['devices'][found_name])
  File "/usr/bin/studio-controls", line 304, in get_dev_info
    sub_db['rate'] = int(rates[0])
IndexError: list index out of range
```
Kubuntu Impish Indri prerelease, Python 3.9.5, Linux 5.9.6. Any ideas, or is this just a bug?

**Comment by CheetahPixie** [comment]
While trying to chase the bug, I found a small inconsistency, Line 856 in /usr/bin/studio-controls contains `rates =[]` as opposed to `rates = []` used everywhere else.

**Comment by CheetahPixie** [comment]
I got it to launch.

I substituted
`sub_db['rate'] = int(rates[0])`
with
`rates.insert(0, sub_db['rate'])`
I have no idea how big of a hack this is, but at least it launches.

**Comment by CheetahPixie** [comment]
I found another solution that won't leak memory:
`rates[0] = sub_db['rate']`
JACK itself, however, refuses to start.

**Comment by CheetahPixie** [comment]
...undoing the fix makes it launch now. Weird.

It seems Jack will still not run, however.

**Comment by ovenwerks** [comment]
It's a bug. I have been developing on the LTS. There are a number of things that have changed from 20.04 to 21.10. I will include your bits of info along with other things :) This is why it is a pre-release. Thank you for your input.

**Comment by CheetahPixie** [comment]
If i could get pointers to where the button to start JACK is in that file/the code, I could probably implement a kill -9 hack to get it started.

**Comment by ovenwerks** [comment]
Please try 2.1.68 first. (we already kill -9 jackd jackdbus at each start) Actually it seems that this device is not giving any list of rates, yicks, I have never run into that before. May not be fixed in 2.1.68 then (I see a 2.1.69 in our future)

**Comment by ovenwerks** [comment]
This should be fixed in 2.1.59
